Dockerfile.rhel7

```diff
@@ -45,6 +48,10 @@ RUN set -x && \
     yum-config-manager --enable rhel-7-server-thirdparty-oracle-java-rpms && \
     yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical --setopt=tsflags=nodocs && \
     yum -y install --setopt=tsflags=nodocs tar java-1.8.0-oracle-devel && \
+    yum-config-manager --enable rhel-7-server-optional-rpms && \
+    ### help markdown to man conversion
+    yum -y install golang-github-cpuguy83-go-md2man && go-md2man -in help.md -out help.1 && \
+    yum -y remove golang-github-cpuguy83-go-md2man && rm -f help.md && \
```

Comment:
`go-md2man` is provided by `golang-github-cpuguy83-go-md2man` it's not a separate package.